index.html

```diff
@@ -1,26 +1,93 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<title>Homework Assignment Three</title>
     	<meta charset="utf-8">
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
-
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
 
 	</head>
 	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
-			</header>
-			<footer>
-				&copy; Footer Goes Here
-			</footer>
+		<div class="topbar clearfix">
+		<br>
```

Comment:
The HTML indentation seems a bit off. Try to write your HTML such that everything is indented 1 tab:

```html
<body>
  <header>
    <h1>Hello</h1>
  </header>
  <footer>
    <h1>World</h1>
  </footer>
</body>
```
This makes the code easier to follow.